gitpod-network-check/cmd/checks.go

```diff
@@ -65,6 +66,10 @@ var checkCommand = &cobra.Command{ // nolint:gochecknoglobals
 			log.Infof("ℹ️  Found duplicate subnets. We'll test each subnet '%v' only once.", distinctSubnets)
 		}
 
+		if networkConfig.ApiEndpoint == "" {
```

Comment:
Previously this parameter was unnecessary, because we were testing if the VPC endpoint for execute-api existed in the context of "this" AWS credential.

As we've learned, this is not always the case, like when a central account is used for VPC endpoints.

So, it became necessary to add this parameter.